README.md

```diff
@@ -40,13 +36,15 @@ All options are optional, and come with defaults. The defaults are shown below:
 ```es6
 headsUp({
   selector: 'header',
+  debounce: false,
```

Comment:
Nit: Inconsistent selector?

Reply:
Huh, it looks the same to me. Maybe it's a tabs/spaces thing? I'll take a look.

Reply:
Would you mind clarifying what you mean here @tylersticka? As far as I can tell all of the properties here are indented the same (2 spaces). I tried dedenting and re-indenting them but it doesn't show up as any changes in git. Thanks!

Reply:
Weird, it went away. I _was_ seeing this in GitHub:

![Screen Shot 2019-06-05 at 8 15 22 AM](https://user-images.githubusercontent.com/69633/58968080-3eb76400-876a-11e9-8eb0-5ce03da6a337.png)

But now I don't. Sorry about that.

Reply:
Oh that is weird! No worries!